facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -29,22 +29,21 @@ object PressedCollectionDeduplication {
 
    */
 
-  def getHeaderURLsFromCuratedAndBackfilled(pCVs: Seq[PressedCollectionVisibility]): Seq[String] = {
+  def getHeaderURLsFromCuratedAndBackfilled(pCVs: Seq[PressedCollectionVisibility], depth: Int): Seq[String] = {
     // Return the header urls of all curated or backfill elements of a sequence of `PressedCollectionVisibility`.
+    // Taken within `depth` of the beginning of the sequence.
 
-    val visibility: Int = 3
-
-    // 11th Dec version:
+    // 11th Dec:
     // To prevent a tiny problem with the Most Popular container I am introducing the effect of collecting only the
-    // first 5 stories of each field. Interestingly the PressedCollectionVisibility has a notion of visibility
+    // first depth stories of each field. Interestingly the PressedCollectionVisibility has a notion of visibility
```

Comment:
Would it be worth expanding a bit more on the concept of `depth` here? Does a higher number mean we'll remove duplicates more aggressively?

Reply:
Yes